index.html

```diff
@@ -120,18 +119,20 @@ <h2 class="h2-work text-color-1 no-margin-bottom">
             <div class="flex-center">
               <h2 class="work-title incard-text-color">Multi-Post Stories Gain+Glory</h2>
             </div>
-            <!--skils-->
+
```

Comment:
- [ ] kindly it would be better for the content of the project to be aligned with the **flexbox** and with [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), you can create complex layouts without having to use floats or positioning, which can make your code more difficult to maintain. 👍